docs/src/architecture/domain-model.md

```diff
@@ -0,0 +1,214 @@
+    * has one timestamp
+    * releates to one **Metric**
+  * **Metric**, is a time-series of measurements
+    * contains a _source device_
+    * contains a _type name_
+    * optionally contains _units_ for the **measurements**
+  * TODO: Example for metric and it's measurements
+  * **Command**, is a single value or set of values
+    * is send from the cloud to one device, e.g. to
+      - stimulate an actuator (e.g. switching a relay)
+      - send a signal to the domain application
+      - set one or more set-points (e.g. upper/lower limits or threshold of a climate control)
+    * values could be a mix of numbers, strings or booleans
+  * **Event**, is a notification that something happened on the device's environment or software system
+    * it's source could be
+      - a sensor that detected a door has beed closed
+      - a signal from the device's domain application
+      - a device's system notification that a user has started an ssh session
+    * has one timestamp given by the producer, or implicitly set as the current time of receiving at **thin-edge**
+  * **Alarm**, is similar to an event, but the _End User_ (an operator of the system) has to take action to resolve the alarm.
+
+### Device Management
+
+**Device Management** provides to manage and monitor devices in the field from the cloud. That includes:
+  * **Software Management**, provides to manage the installed software packages on the device
+    * view list and versions of installed packages
+    * install new or update existing software packages
+    * remove installed software packages
+  * **Configuration Management**, provides to view and change configurations on the device
+    * lists available configuration files
+    * retrieve content of individual configuration files
+    * send new content for individual configuration files
+  * **Log Management**, provides to view log files from the device
+    * lists available log files
+    * retrieve content of individual log files
+  * **Device Monitoring**, collects metrics from the device and forward these to the IoT cloud
+    * allows monitors the health of devices
+    * helps to troubleshoot when problems with the device are reported
+
+## thin-edge device concept
+
+**thin-edge** facilitates IoT functionality to the device it is running on, as well as to devices that are connected to that device.
+
+  * the device thin-edge is running on is referred as the **main-device**
+    * thin-edge on the **main-device** establishes and manages all communication to the cloud
+  * all devices connected to the **main-device** are referred as **external child-devices**
+  * each device is represented in the cloud with an individual **device twin**
+  * a unique **child-id** makes the association between each **external child-device** and it's **device twin**
+  * all telemetry data (i.E. Measurements, Variables, Events, Alarms) of a device (i.E. **main-device** or **external child-devices**) appear
```

Comment:
"Variable" is a new term

Reply:
```suggestion
  * all telemetry data (i.E. Measurements, Commands, Events, Alarms) of a device (i.E. **main-device** or **external child-devices**) appear
```

Thanks for the hint. The term "variable" was replaced with "command", and that line it was kept by mistake.